### DISCUSSION TITLE
AI著作權

### DISCUSSION THREAD

**Discussion by ying0607** [1101-GS3073-C/Python-Community-GS3073-C#652] [General]
最近AI製圖又開始引發討論
甚至AI畫的圖也能在美術展獲獎了
這樣這張圖會有著作權嗎
如果有的話應該要給這個AI軟體還是操作AI的人
https://www.4gamers.com.tw/news/detail/54911/human-uses-ai-to-win-art-competition-fooling-judges-and-angering-actual-artists

**Comment by bww0401** [comment]
隨著科技進步，AI確實達到很多我們認為只有人類做得到的事情，也因此出現了很多爭議，就像AI會幫人做論文一樣，如果沒有制定相關法律與規範，只會出現更多問題

**Comment by larry40305** [comment]
要訓練出一張好看的圖也需要大量的時間跟資料，所以我覺得可以在藝術比賽中特別開一個項目給AI繪圖。

**Comment by 0212Jerry** [comment]
我認為如果真的需要著作權的話，著作權應該屬於操作Ai的人。因為操作的人是依自己的主觀意識在輸入參數，並且依自己的喜好選擇，而AI生成的圖片是經過演算的，並不是通過AI本身的意識，AI也沒有選擇，只有照著演算出來的結果呈現而已。另外，我也認同在比賽中新增人工智慧類別，因為這同樣也是創作，但前提主辦單位有能力辨別機器與人類的作品，不過這個要普及的話目前有難度，包括辨識的工具以及標準等等。期待數年後可以見到。

**Comment by Yilin20021016** [comment]
我覺得AI的開發人員有一小部分的 但大部分是在使用者上

**Comment by Valentinaa0214** [comment]
我認同著作權應該要是給出指令的使用者，一張好的畫作需要使用者在輸入提示資源給AI時精確且經過雕琢。AI繪畫的風潮若是繼續延續下去，這勢必會變成一種新型態結合科技的藝術形式。
